packages/web/README.md

```diff
@@ -77,7 +77,7 @@ const UrgeWithPleasureComponent = () => (
     }}
     isPlaying
     duration={10}
-    colors={[['#A30000']]}
+    colors={[['#A30000',0]]}
```

Comment:
Here the duration should be just `1`: This one looks a bit weird so I am planning to extend this prop and allow adding just a single string as a color, which means there won't be any color animation.